core/src/main/java/edu/wpi/grip/core/events/EventLogger.java

```diff
@@ -0,0 +1,23 @@
+package edu.wpi.grip.core.events;
+
+import com.google.common.eventbus.Subscribe;
+import com.google.inject.Singleton;
+
+import java.util.logging.Logger;
+
+/**
+ * Class for logging events as they're posted to the event bus.
+ */
+@Singleton
+public class EventLogger {
+
+  private static final Logger logger = Logger.getLogger(EventLogger.class.getName());
+
+  @Subscribe
+  public void eventPosted(LoggableEvent event) {
```

Comment:
I'm not going to add the `@AllowConcurrentEvents` annotation; the event bus is synchronous so only one event will be posted at a time anyway.